### DISCUSSION TITLE
[solved-operator error] help with my k150 and PIC16F630

### DISCUSSION THREAD

**Discussion by mer2329** [sanni/cartreader#547] [Need help]
this is my first try with the K150, so im not sure if its operator error, a board error, or bad chips.
ive gotten past the driver issue, and microbrn has stopped failing its resets. so yay to that. 
this is my first try with the K150.

now when i go to program the PIC16F630 i ordered from digikey (they were sold out of the PIC12F629 and i wanted to program myself)
im using supercic-lock_p16f630.hex like the CIC page says

and for my troubles all i get are errors. 
i dont think this is an error, but im saying yes to it.
![image](https://user-images.githubusercontent.com/6768243/194188098-8f868a6e-8978-4abc-b4a1-62925094a594.png)
i confirm i want to change it, and i get another error
![image](https://user-images.githubusercontent.com/6768243/194188150-da7abcc3-f459-400d-afb0-6ba6d1e7e8ba.png)
now i cant get past this error.

heres my software setup
![image](https://user-images.githubusercontent.com/6768243/194189099-5bd23bc5-1fc8-4cb4-b45b-50e536237258.png)

the chip is connected like the software shows,
![20221005_193953](https://user-images.githubusercontent.com/6768243/194189161-ca5a65d0-f5ef-4f58-8fe0-697813996cf7.jpg)
and heres a closeup of the chip
![20221005_194040](https://user-images.githubusercontent.com/6768243/194189172-9287bdde-89dc-4b59-8bf0-d9f83e1913f0.jpg)

my method of insertion is with lever lifted, slot the chip in, and lower lever.

ive gotten the drivers and software from this thread https://forum.hobbycomponents.com/viewtopic.php?f=86&t=1344&p=2785&hilit=k150#p2785

i dont know if theres any logs i can post but please help.

**Comment by sanni** [comment]
![shjsfhsghsgsgsg](https://user-images.githubusercontent.com/19169494/194266973-30c5d58e-1572-421f-b60e-6a539b607fbd.jpg)

**Comment by mer2329** [comment]
thanks a TON, i did have the space indicated by the software, but i was using the lever as a indicator for pin 1.....
it worked i didnt break the chips.

**Comment by mer2329** [comment]
i just realised that i was so stuck on the lever shows orientation i forgot rule number 1 in computers, "upright text shows orientation"

**Comment by Krystman** [comment]
I'm using the same flasher. But I have the PIC12F629 IC. I'm also getting the OSCAL error message. But if I say "yes" the chip flashes ok. However, I still have issues with SA1 carts so I'm trying to debug my build. Is this OSCAL error message normal? Should you answer this with yes?
![IMG_3586](https://github.com/sanni/cartreader/assets/10092987/d2879bf6-2e75-4cbb-8894-318ebecff862)

**Reply by Krystman** [reply]
Never mind. Got it to work. Which means I assume the error message is normal and can be answered with "yes".

Unrelated tip: clean your cart contacts yo!
